test/evergreen.yml

```diff
@@ -2407,6 +2407,151 @@ tasks:
             cp ../../../bench/wtperf/split_heavy.wtperf .
             ./wtperf -O ./split_heavy.wtperf -o verbose=2
 
+  - name: data-validation-stress-test-checkpoint
+    depends_on:
+    - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix/test/checkpoint"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+
+            CMD='nice ../../../test/checkpoint/recovery-test.sh "-t r -W 3 -D -n 100000 -k 100000 -C cache_size=100MB" WT_TEST.$t'
+
+            NPROCS=$(echo "`grep -c ^processor /proc/cpuinfo` / 2" | bc)
+
+            outf=./outfile.txt
+            for i in $(seq 120); do
+              echo "$d: Iteration $i" >> $outf
+              echo "$d: Iteration $i"
+              for((t=0; t<$NPROCS; t++)); do
+                > nohup.out.$t ; eval nohup $CMD > nohup.out.$t 2>&1 &
+              done
+
+              for((t=0; t<$NPROCS; t++)); do
+                wait -n || exit $?
+              done
+            done
```

Comment:
Can this section be replaced by the new `run_parallel.sh` script?

Reply:
Fixed.